build-tools/debian-metadata/copyright

```diff
@@ -0,0 +1,171 @@
+Xamarin.Android SDK
```

Comment:
Good point. It started as a copy because I started off working from a bundle zip download, and didn't think to decouple it again when re-integrating with the build system.